ui/temboardui/cli/register_instance.py

```diff
@@ -91,14 +91,14 @@ def main(self, args):
         except Exception as e:
             raise UserError(str(e))
 
-        agent = "%s:%s" % (args.agent_address, args.agent_port)
+        agent = f"{args.agent_address}:{args.agent_port}"
 
         session = Session()
         logger.debug("Check for existing instance.")
         instance = get_instance(session, args.agent_address, args.agent_port)
         if instance:
             if not args.skip_existing:
-                raise UserError("Instance %s:%s already registered." % (
+                raise UserError("Instance {}:{} already registered.".format(
```

Comment:
Can't we use f-string here?

Reply:
better keep % formatting in this error handling